client/src/components/layouts/default.tsx

```diff
@@ -16,7 +17,8 @@ import { isBrowser } from '../../../utils';
 import {
   fetchUser,
   onlineStatusChange,
-  serverStatusChange
+  serverStatusChange,
+  loadAllChallengesInfo
```

Comment:
```suggestion
  updateAllChallengesInfo
```
We should be consistent with `updateChallengeMeta` since both take info from Gatsby and put it into the store.

Reply:
addressed in a separate commit.